testing.md

```diff
@@ -0,0 +1,71 @@
+# Testing
+
+## Why write tests?
+
+We write automated tests to allow us to quickly demonstrate that our software meeting business requirements. They're one of a few tools we use to increase our confidence in our work prior to deployment.
```

Comment:
I assume your disagreement is with the, "confidence in our work prior to deployment" rather than the sentence before? When I wrote that, I meant "confidence in our work" as "confidence in our software meeting business requirements now and in the future", not confidence that our code does what we want it to. Business requirements encompasses regressions, speed, security, functionality etc?